decidim/decidim
# Fix the Diff Render output

#### :tophat: What? Why?
While working on #13396, i have noticed the format of the title is wrong. Investigating this, i have understood what was happening and I fixed it. 

Since in machine translated installations, the locale is being displayed as a Hash it becomes very hard to properly understand what is happening, i have split also the machine translated field in own field, as per last screenshot. 

![image](https://github.com/user-attachments/assets/fec309bb-8c79-4a80-8f8e-505c98782c71)

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #13396 

#### Testing
1. Visit any proposal change set page ( like version 1) 
2. See the way Title and Body are being displayed
3. Apply patch 
4. See the new way 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![image](https://github.com/user-attachments/assets/fc6165a4-d6e2-498f-b872-de7217be06fc)

![image](https://github.com/user-attachments/assets/fbefcc38-f75a-451f-b8c1-e334d43fc911)

Last form 
![image](https://github.com/user-attachments/assets/4d0be95d-824e-43ac-9219-c2a1298bbd74)


:hearts: Thank you!
